lib/context.js

```diff
@@ -58,6 +61,28 @@ class Context {
   get isBot() {
     return this.payload.sender.type === 'Bot';
   }
+
+  /**
+   * Reads the plugin configuration from the given YAML file in the `.github`
+   * directory of the repository.
+   *
+   * @example
+   *
+    // Load config from .github/myplugin.yml in the repository
+    const config = await context.config('myplugin.yml');
+
+    if(config.close) {
+      context.github.issues.edit(context.issue({state: 'closed'}));
+    }
+   *
+   * @param {string} fileName Name of the YAML file in the `.github` directory
+   * @return {Promise<Object>} Configuration object read from the file
+   */
+  async config(fileName) {
+    const params = this.repo({path: path.join('.github', fileName)});
```

Comment:
Yeah, I like that it's opinionated. If there are existing config files outside of `.github`, there's also a good chance that they're not `yml` (e.g. [OWNERS](https://github.com/probot/owners/blob/master/lib/owner-notifier.js#L43)), so it can't use this method anyway.

I'm not opposed to making this more configurable later, or even adding a `context.content` method that loads files from the repo, but it's only 2-3 lines of code to load files, so I'd prefer to wait for now.